src/controllers/news.ts

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable @typescript-eslint/type-annotation-spacing */
+/* eslint-disable eol-last */
+/* eslint-disable prettier/prettier */
```

Comment:
Can you remove this statements, you don't need to disable anything, for prettier just running fixes everything automatically `npm run prettier-fix`. Let's follow these style guides strictly